README.md

```diff
@@ -7,143 +7,33 @@
 
 <img width="100" align="left" src="logo.png">
 
-This is a work-in-progress
-[Brigade 2](https://github.com/brigadecore/brigade/tree/v2)
-compatible gateway that receives events 
+The Brigade Bitbucket Gateway receives events 
```

Comment:
```suggestion
The Brigade Bitbucket Gateway receives event 
```

Reply:
Oh, yup, looking closer I see it is "The Brigade Bitbucket Gateway receives events (webhooks)...", not "The Brigade Bitbucket Gateway receives events webhooks..." like I had thought :)  Indeed, please disregard.

Reply:
No worries!